packages/lit-html/package.json

```diff
@@ -155,6 +152,127 @@
     "!/development/test/",
     "/directives/"
   ],
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts:types",
+        "build:version-stability-test"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo",
+        "tsconfig.polyfill-support.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "directives/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "async-directive.js{,.map}",
+        "directive-helpers.js{,.map}",
+        "directive.js{,.map}",
+        "experimental-hydrate.js{,.map}",
+        "lit-html.js{,.map}",
+        "polyfill-support.js{,.map}",
+        "private-ssr-support.js{,.map}",
+        "static.js{,.map}",
+        "directives/*.js{,.map}"
+      ]
+    },
+    "build:version-stability-test": {
+      "command": "rollup -c rollup-version-stability-test.config.js",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup-version-stability-test.config.js",
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "version-stability-build"
+      ]
+    },
+    "checksize": {
+      "command": "rollup -c --environment=CHECKSIZE",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": []
+    },
+    "check-version": {
+      "command": "node scripts/check-version-tracker.js",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "scripts/check-version-tracker.js",
+        "package.json",
+        "src/lit-html.ts"
+      ],
+      "output": []
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "command": "cd ../tests && MODE=dev npx wtr \"../lit-html/development/**/*_test.(js|html)\"",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "../tests/web-test-runner.config.js",
```

Comment:
Could a Wireit script hand only files so you could use it as a dependency to get this effect and be robust to the WTR config being split into more files?

Reply:
Yes, actually in my prototype I did this :) https://github.com/aomarks/wireit-lit-fork/blob/1782c67d2a123106a5f73bbc46c962221e151121/packages/tests/package.json#L41

However, I forgot about this and implemented a validation check that requires all scripts to have either a `command` or `dependencies`: https://github.com/google/wireit/blob/7cf075e186de9bc06e5f7e0da3e6d39fdd50daba/src/analyzer.ts#L215

Added a `#comment`.

(Also realized I forgot to include `rollup-resolve-remap.js` there too).